### PR TITLE
SYS-1585 added VER variable

### PR DESCRIPTION
**Pull request by jakeiotechsys**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -44,7 +51,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       subconfig.vm.network :private_network, ip: "192.168.33.#{i + 10}"
     
       # Provision VMs using a provision (shell) script
-      subconfig.vm.provision :shell, path: "node-bootstrap.sh"    
+      subconfig.vm.provision :shell, path: "node-bootstrap.sh"
```
Comment (rbose21) [inline Vagrantfile:54]: This need the version argument too

Comment (jakeiotechsys) [reply Vagrantfile:54]: Yeah, I was just testing if the syntax would work properly on the server first!

File: master-bootstrap.sh [hunk 2]
```diff
@@ -1,17 +1,17 @@
 #!/usr/bin/env bash
-
+VER=1.1.0
```
[the remark this answers is not in the document]
Comment (jakeiotechsys) [reply master-bootstrap.sh]: Great point! I've made some changes to the scripts to make it more streamlined. We only have 1 place where the ver is set now

**Review by rbose21 — COMMENTED** [review]
LGTM

**Review by rbose21 — APPROVED** [review]
LGTM

**Pull request merged**